datanode/networkhistory/snapshot/service_load_snapshot.go

```diff
@@ -256,7 +262,7 @@ type compressedFileMapping interface {
 }
 
 func (b *Service) loadSnapshot(ctx context.Context, loadLog LoadLog, snapshotData compressedFileMapping, copyFromDirectory string,
-	dbMetaData DatabaseMetadata, vegaDbConn *pgxpool.Pool, withIndexesAndOrderTriggers bool,
+	dbMetaData DatabaseMetadata, vegaDbConn *pgxpool.Pool, withIndexesAndOrderTriggers bool, historyTableLastTimestamps map[string]time.Time,
```

Comment:
I don’t quite get why we have `vegaDbConn *pgxpool.Pool` as a parameter here (and in other methods of `*Service`), and not use `b.connPool`?

Reply:
Good point, it was something inherited from the original implementation and I have now fixed.